include/xtensor/xoperation.hpp

```diff
@@ -959,6 +959,74 @@ namespace xt
     {
         return detail::make_xfunction<typename detail::cast<R>::functor>(std::forward<E>(e));
     }
+
+    /**
+    * @ingroup logical_operators
+    * @brief isin
+    *
+    * Returns a boolean array of the same shape as element that is ``true`` where an element of
+    * ``element`` is in ``test_elements`` and ``False`` otherwise.
+    * @param e an \ref xexpression
+    * @return a boolean array
+    */
+    template <class E, class T>
+    inline auto isin(E&& e, std::initializer_list<T> values) noexcept
```

Comment:
Argument names should match those of the documentation.

Reply:
Will do, I will just keep NumPy's names for the arguments